TwoDo/app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright (c) Microsoft Corporation. All rights reserved.
+  ~  Licensed under the MIT License.
+  -->
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.example.twodo">
```

Comment:
Not sure if we want to use this package or one under `Microsoft` as we have in other repos.

Reply:
I'll follow the format Raymond used for the eBook sample: com.microsoft.device.wm_samples.twodo